experiment/popularity_community/crawl_torrents.py

```diff
@@ -0,0 +1,214 @@
+        """
+        super().introduction_response_callback(peer, dist, payload)
+        self._logger.debug(f"Peer {peer} wants to introduce {payload.wan_introduction_address}")
+
+
+class Service(TinyTriblerService):
+    def __init__(self, output_file_path, timeout_in_sec, working_dir, config_path):
+        super().__init__(Service.create_config(working_dir, config_path),
+                         timeout_in_sec, working_dir, config_path)
+        self._output_file_path = output_file_path
+
+    @staticmethod
+    def create_config(working_dir, config_path):
+        config = TinyTriblerService.create_default_config(working_dir, config_path)
+
+        config.set_ipv8_enabled(True)
+        config.set_ipv8_walk_interval = IPV8_WALK_INTERVAL
+
+        config.set_chant_enabled(True)
+        return config
+
+    async def on_tribler_started(self):
+        await super().on_tribler_started()
+        session = self.session
+        peer = Peer(session.trustchain_keypair)
+
+        session.remote_query_community_community = Crawler(peer, session.ipv8.endpoint,
+                                                           session.ipv8.network,
+                                                           session.mds, self._output_file_path)
+
+        session.ipv8.overlays.append(session.remote_query_community_community)
+        session.ipv8.strategies.append((RandomWalk(session.remote_query_community_community,
+                                                   timeout=RANDOM_WALK_TIMEOUT_IN_SEC,
+                                                   window_size=RANDOM_WALK_WINDOW_SIZE,
+                                                   reset_chance=RANDOM_WALK_RESET_CHANCE),
+                                        UNLIMITED))
+
+
+def _parse_argv():
+    parser = argparse.ArgumentParser(description='Crawl first 100 torrents from random nodes in the network')
+    parser.add_argument('-t', '--timeout', type=int, help='a time in sec that the experiment will last', default=0)
+    parser.add_argument('-f', '--file', type=str, help='result file path (json)', default='result.json')
+    parser.add_argument('-v', '--verbosity', help='increase output verbosity', action='store_true')
+
+    return parser.parse_args()
+
+
+def _run_tribler(arguments):
+    service = Service(arguments.file, arguments.timeout,
+                      working_dir=os.path.join(
+                          '/tmp/tribler/experiment/popularity_community/crawl_torrents',
+                          '.Tribler'),
+                      config_path='./tribler.conf')
+    loop = asyncio.get_event_loop()
+    loop.create_task(service.start_tribler())
+    try:
+        loop.run_forever()
+    finally:
+        loop.run_until_complete(loop.shutdown_asyncgens())
+        loop.close()
+
+
+if __name__ == "__main__":
+    _arguments = _parse_argv()
+    print(f"Arguments: {_arguments}")
+
+    if _arguments.verbosity:
```

Comment:
Better to rewrite with: `logging.basicConfig(level=logging.DEBUG if _arguments.verbosity else logging.INFO`

Reply:
I don't like ternary operators in python. They bulky.
I'll think.

Reply:
I've rewrite code:

```
    logging_level = logging.DEBUG if _arguments.verbosity else level = logging.INFO
    logging.basicConfig(level=logging_level)
```

thank for the suggestion